### PR TITLE
OPD fixing STIs and additional of drug and diagnosis

### PR DESCRIPTION
**Pull request by petroskayange**
1. fixing OPD STIs
2. Adding OPD diagnosis = Genital herpes,haemorrhoids
3. Adding OPD drug = cenfixine 400mg

### CODE REVIEW

File: db/idsr_metadata/idsr_ohsp_settings.json.example [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"headers":      {"url":"10.40.30.2:8001","user":"user_admin","pass":"pass_district"},
-	"sms_server":   {"url":"10.40.30.2:8004","user":"sms_user","pass":"sms_pass"},
+	"sms_server":   {"url":"10.40.30.11:8186/submit","user":"admin@email.test","pass":"administrator"},
```
Comment (Skipper-116) [inline db/idsr_metadata/idsr_ohsp_settings.json.example]: Are these actual credentials? Is this possible to be moved to the configurations

**Review by Skipper-116 — COMMENTED** [review]
From what i'm seeing we need to stop tracking the idsr_ohsp_settings.json othewise we are tracking credentials on a public repo.

**Review by Skipper-116 — CHANGES_REQUESTED** [review]
Just changes on the idsr_metadata

File: db/idsr_metadata/idsr_ohsp_settings.json.example (hunk 1 above)
Comment (petroskayange) [reply db/idsr_metadata/idsr_ohsp_settings.json.example]: Alright let me move them

**Review by Skipper-116 — APPROVED** [review]

**Pull request merged**